Lib/fontTools/feaLib/parser.py

```diff
@@ -2079,13 +2089,11 @@ def check_glyph_name_in_glyph_set(self, *names):
         If no glyph set is present, does nothing.
         """
         if self.glyphNames_:
-            missing = [name for name in names if name not in self.glyphNames_]
-            if missing:
-                raise FeatureLibError(
-                    "The following glyph names are referenced but are missing from the "
-                    f"glyph set: {', '.join(missing)}",
-                    self.cur_token_location_,
-                )
+            for name in names:
+                if name in self.glyphNames_:
+                    continue
+                if name not in self.missing:
+                    self.missing[name] = self.cur_token_location_
```

Comment:
the docstring of `check_glyph_name_in_glyph_set` says "Raises if..." but it no longer raises, does it?